deploy/policy/users.yml

```diff
@@ -15,9 +15,9 @@
 - !grant
```

Comment:
are all the groups and users in this files used? can we remove those that are not?

Reply:
nope removing